src/content/reference/react/use-client.md

```diff
@@ -23,37 +23,364 @@ canary: true
 
 ### `'use client'` {/*use-client*/}
 
-Add `'use client';` at the very top of a file to mark that the file (including any child components it uses) executes on the client, regardless of where it's imported.
+Add `'use client'` at the top of a file to mark the module and its transitive dependencies as client code.
```

Comment:
This use of "transitive dependencies" is correct but my brain erroneously interpreted it as "transitive dependents" instead. I wonder if we could say something like "and any other modules it imports".

Reply:
I think that weakens the definition though because 'any other module it imports' may be only interpreted as one level